app/views/markets/show.html.erb

```diff
@@ -1,25 +1,23 @@
-<div class="container" style="height:  width:60%" >
+<div class="container">
```

Comment:
this wasn't doing anything because `  width:60%` is not a valid `height` property :)